src/org/opensolaris/opengrok/authorization/AuthorizationStack.java

```diff
@@ -269,6 +275,18 @@ protected boolean processStack(Nameable entity,
                 }
             }
         }
+        time = System.currentTimeMillis() - time;
+
+        stats.addRequestTime(pluginPredicate.getRequest(),
+                String.format("authorization_in_stack_%s_%s", getName(), overallDecision ? "positive" : "negative"),
+                time);
+        stats.addRequestTime(pluginPredicate.getRequest(),
```

Comment:
I am baffled by the `request` argument of `addRequestTime()` which calls `addRequest()` but that is not using the `req` argument either. So why have it at all ?